src/api-docs/cart.docs.js

```diff
@@ -106,32 +40,79 @@ const updateCart = {
       }
     }
   },
-  responses: {
-    200: {
-      description: "Success"
-    },
-    400: {
-      description: "Bad request"
-    },
-    401: {
-      description: "Unauthorized"
-    },
-    500: {
-      description: "internal server error"
+  responses:Responses
+};
+
+const getCartItems = {
+  tags: ["Cart"],
+  description: "list all items in cart",
+  parameters: [],
+  security: [
+    {
+      token: []
     }
-  }
+  ],
+  responses: Responses
```

Comment:
same as here, if you rename import to responses here you can just type `responses,`. The shorthand property syntax to simplify the object literal notation when creating an object with a property name that matches a variable name.